src/components/CDate/CDate.js

```diff
@@ -109,6 +109,15 @@ export default {
 
     const data = {
       props: getMenuProps(this),
+      on: {
+        input(value) {
+          if (value) {
+            self.sendToEventBus('Opened', { value });
```

Comment:
Can we have one event here called VisibilityChanged with payload `visible: true/false`. Goal is to have smaller number of events is possible.